specutils/analysis/width.py

```diff
@@ -149,13 +149,13 @@ def _compute_single_fwhm(flux, spectral_axis):
     right = flux[argmax+1:] <= halfval
 
     # Highest signal at the first point
-    if len(left) == 0:
+    if sum(left) == 0:
```

Comment:
```suggestion
    if np.sum(left) == 0:
```

Here (and below) I think you want to use `np.sum` - it's usually more performant for boolean arrays (and in principle this array is ~as big as the spectrum, right?